library/spdm_requester_lib/libspdm_req_set_certificate.c

```diff
@@ -0,0 +1,188 @@
+/**
+ *  Copyright Notice:
+ *  Copyright 2021-2022 DMTF. All rights reserved.
+ *  License: BSD 3-Clause License. For full text see link: https://github.com/DMTF/libspdm/blob/main/LICENSE.md
+ **/
+
+#include "internal/libspdm_requester_lib.h"
+
+#pragma pack(1)
+typedef struct {
+    spdm_message_header_t header;
+} libspdm_set_certificate_response_max_t;

```

Comment:
why we need this? it is same as spdm_set_certificate_request_t

Reply:
Thanks. I have changed it.